lib/pbench/test/unit/server/query_apis/test_namespace_and_rows.py

```diff
@@ -351,6 +349,7 @@ def test_query(
             json=response_payload,
             status=HTTPStatus.OK,
             headers=build_auth_header["header"],
+            request_method=API_METHOD.GET,
```

Comment:
I'm stumped:  how do I know whether this is right?

Previously, we had `self.api_method` here, which seemed nice and flexible.  This is fine if it's right, but...how can I tell?

Reply:
Since this is a specific test case, it knows what it needs. At the same time, it's true that this change was unnecessary since the previous `self.api_method` reference would do exactly the same thing. Siddardh removed the class `api_method` attribute in this case, but the `Commons._setup` will still set an *instance* `api_method` attribute based on the test class schema.

The `self.api_method` is really intended for use within `Commons` which has to support both `GET` and `POST` ... the specific test classes don't need to be generalized, but certainly *could* depend on the same attribute. And it's arguably a bit easier to read with the explicit value here.

Reply:
> Since this is a specific test case, it knows what it needs.

Yes.  But, as a reviewer, I'm supposed to decide whether what it professes to know is actually correct.  😛

In any case, since you seem to think that `GET` is the correct value, I shall be satisfied.  (Thanks for the explanation.)